### ISSUE TITLE
Runtime translations applying is late

### ISSUE THREAD

**Issue by npub** [Sequel-Ace/Sequel-Ace#1226]
- Sequel Ace Version: 1.60.1 (83bd43bc519d15e50c4272c6cf5c1479df196a4d)
- Sequel Ace Source: App Store
- macOS Version: 11.6 (20G165)
- Processor Type: Intel
- MySQL Version: 8.0.26
- macOS Localization: Russian

**Description**
Runtime translations applying to menu is late or internal trigger fails.

**Steps To Reproduce**
1. Restart App.
2. Connect to DB.
3. Open Table menu — it is not translated.
4. Choose any table (at App`s left side).
5. Open Table menu again — it`s translated now.

**Expected Behaviour**
The menu and other elements of GUI should be translated immediately after the application is launched.

**Comment by Kaspik** [comment]
Is this still happening?

**Comment by npub** [comment]
Yes. Sequel Ace v.3.5.2 Build 20033 (from App Store).

Step 3:
![before (step 3)](https://user-images.githubusercontent.com/751867/183856574-61c901e2-f5a1-4655-8f9f-0f82e1103d44.png)

Step 5:
![after (step 5)](https://user-images.githubusercontent.com/751867/183856560-66c43715-3643-49a1-9cd5-e8db76030a89.png)

Also seems like some menu`s titles does not exists at https://crowdin.com/project/sequel-ace project to be translated.

**Comment by Kaspik** [comment]
Yeah that's the problem with xib / storyboard. We are not translating UI yet, so it could be the same issue with these late-applied translations 😞
